reactions/src/SPT.cpp

```diff
@@ -749,6 +790,32 @@ switch (a) {
         return 0;
 }}
 
+// BILL MOD
+double SPT::PLOOPn2_nu(int a, double vars[], double k, double err) const{
+  IOW iow;
+double loop, tree;
+double prefac = k*k*k/(4*M_PI*M_PI);
+real KMAX = QMAXp/k;
+real KMIN = QMINp/k;
+double c[2] = {KMIN,-1.};
+double d[2] = {KMAX, 1.};
+switch (a) {
+  case 0:
+    iow.initn_lin(vars[0], k, vars[1],vars[2], vars[3],vars[4]);
```

Comment:
Don't need the initn_lin anymore since P_l already has growth

Reply:
Ok thanks, have removed it now.